### PR TITLE
Fix Next Steps & Fix README - resolve #198 

### LINKED ISSUE #198: 第一次运行提示错误
### 服务端版本 LeavesMC Version

git-Leaves-"a4a3a92" (MC: 1.20.4)

### 预期的行为 Expected behavior

应提示阅读Leaves文档

### 观测到的实际行为 Observed/Actual behavior

 [Server thread/INFO]: Paper: Using libdeflate (Linux x86_64) compression from Velocity.
 [Server thread/INFO]: Paper: Using OpenSSL 3.0.x (Linux x86_64) cipher from Velocity.
*************************************************************************************
[Server thread/INFO]: This is the first time you're starting this server.
[Server thread/INFO]: It's recommended you read our 'Getting Started' documentation for guidance.
[Server thread/INFO]: View this and more helpful information here: https://docs.papermc.io/paper/next-steps
*************************************************************************************

### 复现步骤 Steps/models to reproduce

第一次运行服务端即可

### 插件与数据包列表。 Plugin and Datapack List.

无

### 更多详细信息。 The more information.

_No response_

### PR DESCRIPTION
**Pull request by silvery0113**
resolve #198

### CODE REVIEW

**Comment by LittleChest** [comment]
设置一下关联

**Comment by Bluemangoo** [comment]
- resolve #198

File: patches/server/0136-Fix-next-steps.patch
Comment (s-yh-china) [inline patches/server/0136-Fix-next-steps.patch:1]: 看起来不像是 `rebuildPatches` 生成的，我建议重新生成一下patch

**Review by s-yh-china — CHANGES_REQUESTED** [review]

**Review by s-yh-china — APPROVED** [review]

**Comment by s-yh-china** [comment]
thanks!

**Pull request merged**